### PR TITLE
Only sign and publish once at the join point

### PR DESCRIPTION
**Pull request by mmitche**
All the artifacts are present, so we only need to sign once

### CODE REVIEW

**Review by steveisok — APPROVED** [review]

File: eng/azure-pipelines.yml [hunk 1]
```diff
@@ -291,12 +291,7 @@ extends:
...
-                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:PackageRID=osx-arm64 /p:AssetManifestOS=osx /p:PlatformName=arm64 $(_InternalBuildArgs)
-                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:PackageRID=linux-x64 /p:AssetManifestOS=linux /p:PlatformName=x64 $(_InternalBuildArgs)
-                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:PackageRID=linux-arm64 /p:AssetManifestOS=linux /p:PlatformName=arm64 $(_InternalBuildArgs)
-                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:PackageRID=linux-musl-x64 /p:AssetManifestOS=linux-musl /p:PlatformName=x64 $(_InternalBuildArgs)
-                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:PackageRID=linux-musl-arm64 /p:AssetManifestOS=linux-musl /p:PlatformName=arm64 $(_InternalBuildArgs)
+                  ./eng/common/build.ps1 -ci -configuration $(_BuildConfig) -restore -sign -publish /p:AssetManifestOS=unix-all $(_InternalBuildArgs)
```
Comment (akoeplinger) [inline eng/azure-pipelines.yml:294]: do we need to set AssetManifestOS? `unix-all` doesn't really exist so it might be confusing

**Review by akoeplinger — APPROVED** [review]

**Review by akoeplinger — APPROVED** [review]

**Comment by mmitche** [comment]
https://dev.azure.com/dnceng/internal/_build/results?buildId=2409981&view=results

**Comment by akoeplinger** [comment]
@mmitche that's a dotnet/runtime build and the failure is https://github.com/dotnet/runtime/issues/53420

**Review by lewing — APPROVED** [review]

**Comment by mmitche** [comment]
> @mmitche that's a dotnet/runtime build and the failure is [dotnet/runtime#53420](https://github.com/dotnet/runtime/issues/53420)

doh https://dev.azure.com/dnceng/internal/_build/results?buildId=2409997&view=results

**Pull request merged**